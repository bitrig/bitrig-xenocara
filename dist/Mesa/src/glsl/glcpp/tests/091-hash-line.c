#line 0
#error line 0 error
#line 25
#error line 25 error
#line 0 1
#error source 1, line 0 error
#line 30 2
#error source 2, line 30 error
