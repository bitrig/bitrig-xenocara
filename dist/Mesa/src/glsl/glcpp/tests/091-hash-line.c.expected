0:0(1): preprocessor error: #error line 0 error
0:25(1): preprocessor error: #error line 25 error
1:0(1): preprocessor error: #error source 1, line 0 error
2:30(1): preprocessor error: #error source 2, line 30 error









